app/services/get-export-csv.js

```diff
@@ -0,0 +1,116 @@
+const organisationUnitConstants = require('../constants/organisation-unit')
+const getOrganisationUnit = require('./helpers/org-unit-finder')
+const json2csv = require('json2csv')
+const tabs = require('../constants/wmt-tabs')
+
+const CASELOAD_FIELDS = ['name', 'gradeCode', 'totalCases', 'untiered', 'd2', 'd1', 'c2', 'c1', 'b2', 'b1', 'a']
+const OM_OVERVIEW_FIELDS = ['grade', 'teamName', 'capacity', 'cases', 'contractedHours', 'reduction']
+const OM_OVERVIEW_FIELD_NAMES = ['GradeCode', 'TeamName', 'CapacityPercentage', 'TotalCases', 'ContractedHours', 'ReductionHours']
+const ORG_OVERVIEW_FEILDS = ['name', 'capacityPercentage', 'availablePoints', 'contractedHours', 'reductionHours', 'totalCases']
+
+module.exports = function (organisationLevel, result, tab) {
+  var filename = getFilename(result.title, tab)
+
+  var fieldsObject = getFields(organisationLevel, tab)
+  var fields = fieldsObject.fields
+  var fieldNames = fieldsObject.fieldNames
+
+  var csv = getCsv(organisationLevel, result, tab, fields, fieldNames)
+
+  return { filename: filename, csv: csv }
+}
+
+// TODO: Do we have an agreed naming scheme they would like for these csvs? Org level? Date?
+var getFilename = function (orgName, screen) {
+  var replaceSpaces = / /g
+  return (orgName + ' ' + screen + '.csv').replace(replaceSpaces, '_')
+}
+
+var getFields = function (organisationLevel, tab) {
+  var childOrgForFieldName
+  var fields
+  var fieldNames
+
+  switch (tab) {
+    case tabs.CASELOAD:
+      childOrgForFieldName = getChildOrgForFieldName(organisationLevel)
+      fields = CASELOAD_FIELDS
+      fieldNames = [childOrgForFieldName + 'Name', 'Grade', 'Overall', 'Untiered', 'D2', 'D1', 'C2', 'C1', 'B2', 'B1', 'A']
```

Comment:
Still building up the fieldNames, should this array not be one of your consts above?